client/js/widgets/seat.js

```diff
@@ -33,6 +33,10 @@ class Seat extends Widget {
       displayedText = displayedText.replaceAll('playerName',this.get('player'))
       setText(this.domElement, displayedText);
     }
+    const scoreboard = widgetFilter(w => w.get('type') == 'scoreboard');
+    for (const board of scoreboard)
```

Comment:
Seems like this should be part of updateLinkedWidgets. Also, it seems like this shouldn't need to unconditionally update the scoreboards but only need to update if certain properties are changed.

If in a single delta you update multiple seats this will lead to completely rebuilding all the scoreboard tables for every seat update which seems inefficient. Perhaps we could track whether any seats have updated and then explicitly update the scoreboard after all of the seat updates?

Reply:
While all of this is true, I don't see somebody spamming seat updates. We could even only update certain cells if the score changes but I feel like this is wasted effort until this actually becomes a performance issue somewhere.

Reply:
Yeah I considered updating only the relevant cell as well, but passed on it as being more work than it's worth. Happy to ignore this inefficiency for now until it turns into another `svgReplaces`, though it would be worth commenting that this will rebuild the scoreboard for all seats for each seat update for future reference as a possible optimization.